dependencies.yml

```diff
@@ -0,0 +1,4 @@
+
+- groupId: "org.pih"
+  artifactId: "openmrs-config-pihemr"
+  version: "1.0.0-SNAPSHOT"
```

Comment:
So the Maven dependencies are not part of the `pom.xml`? This is because of how one of these plugins work? Is that documented?

Reply:
It is documented in the README of the plugin.  We'll need to figure out how to publish that documentation effectively going forward.

Reply:
The README is a great place for that documentation. It might be nice to link to it from a confluence page that describes the system more generally.

Reply:
Yes, one of my outstanding to do before I close this ticket is to make sure I document and organize everything properly... we will need to update confluence and Mirebalais README with how to set up a dev environment, among other things.